src/include/parser/parse_node.h

```diff
@@ -89,6 +89,7 @@ typedef enum ParseExprKind
  */
 typedef Node *(*PreParseColumnRefHook) (ParseState *pstate, ColumnRef *cref);
 typedef Node *(*PostParseColumnRefHook) (ParseState *pstate, ColumnRef *cref, Node *var);
+typedef List *(*PostParseFunctionRefHook) (ParseState *pstate, FuncCall *fn, List *fargs);
```

Comment:
i wonder if the naming is correct, `PostParseFunctionRefHook`. since we aren't technically calling the hook post func processing, it's before the call to `ParseFuncOrColumn`.

Reply:
It is default convention we have also followed it in transformColumnRef function

Reply:
IMO, this should actually be `PreParseFunctionRefHook` since this steps precedes the actual function parsing logic.

Reply:
Done